docs/apps/r-env.md

```diff
@@ -0,0 +1,133 @@
+# R
+
+R is an open-source language and environment for statistical computing and graphics. More information on R can be found on the [R Project website](https://www.r-project.org/about.html). A useful set of R manuals is also hosted on [CRAN](https://cran.r-project.org/manuals.html).
+
+## Available
+
+The `r-env` module module has been built for diverse use cases and includes 800+ pre-installed R packages, including support for geospatial analyses. Several [Bioconductor](https://www.bioconductor.org/) packages are also included. Bioconductor is an open-source project providing tools for the analysis of high-throughput genomic data.
+
+Currently supported R versions:
+
+- 3.6.1
+
+Currently supported Bioconductor versions:
+
+- 3.9 (with R 3.6.1)
+
+Other software included in the module:
+
+- GRASS 7.8.0 and Saga-GIS 7.3.0 (with R 3.6.1)
+
+## Licenses
+
+Information on licenses that are in use for R and associated software (such as packages) can be found on the [R Project website](https://www.r-project.org/Licenses/).
+
+## Usage
+
+#### Loading the module
+
+To use the default version of this module on Puhti, initialize it with:
+
+```
+module load r-env
+```
+
+Note that Puhti login nodes are not intended for heavy computing. To use R in Puhti, please either request an interactive job in a computing node or submit a non-interactive batch job in Slurm. 
+
+#### Interactive use
+
+To interactively use R on Puhti's computational nodes, run this command after initializing the `r-env` module. This will request a single-processor R session. Note that you will need to modify the requested duration, memory, project ID and partition as required:
+
+```
+srun -n1 -t hh:mm:ss --x11=first --mem=4G --pty --account=project_id -p partition R --no-save 
+```
+
+For information on available partitions, see [here](https://github.com/CSCfi/csc-user-guide/blob/master/docs/computing/running/batch-job-partitions.md).
```

Comment:
`https://github.com/CSCfi` ... links are not valid page links as they just go to the github files and not a proper page.